CMakeLists.txt

```diff
@@ -212,7 +212,7 @@ elseif (("${CMAKE_C_COMPILER_ID}" STREQUAL "GNU") OR ("${CMAKE_C_COMPILER_ID}" S
 	set(CMAKE_CXX_FLAGS_RELEASE "${CMAKE_CXX_FLAGS_RELEASE} -O3")
 
 	# enable somewhat modern C++
-	set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11")
+	set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=gnu++11")
```

Comment:
Looks like `isnan()` [isn't declared with mingw-w64](https://travis-ci.org/smcv/OpenJK/jobs/87627720). The compiler error suggests `std::isnan`, but `q_shared.h` needs to be valid C so that `codemp` can use it, so that would require a `#ifdef __cplusplus` or something.